pages/portal/chapters/create.tsx

```diff
@@ -0,0 +1,152 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import { addChapter } from "requests/Chapter";
+import { Chapter } from 'utils/types';
+
+import Navigation from "components/Portal/Navigation";
+
+const handleSubmit = async (e) => {
+    e.preventDefault();
+    const formData = new FormData(e.target);
+    var chapter: Chapter = await addChapter(formData);
+    //After submitting the form send the user to the chapter edit page
+    var chapterName = String(formData.get("name"));
+    window.location.href = chapterName?.replace(/ /g, "_")
+};
+
+const Chapters: NextPage = () => {
+    return (
+        <div className="container">
+            <Head>
+                <title>MindVersity | Admin Portal</title>
+                <link rel="icon" href="/favicon.ico" />
+            </Head>
+
+            <Navigation />
+
+            <div className="bodyContent">
+                <h1>New Chapter</h1>
+                <div className="formContainer">
+                    <form onSubmit={handleSubmit} method="post">
+                        <label htmlFor="name">Chapter Name</label>
+                        <input type="text" name="name" placeholder="Chapter Name" required/>
+                        <label htmlFor="region">Region</label>
+                        <select name="region" id="">
+                            <option value="Northeast">Northeast</option>
+                            <option value="South">South</option>
+                            <option value="West">West</option>
+                            <option value="Midwest">Midwest</option>
+                        </select>
+                        <label htmlFor="city">City</label>
+                        <input type="text" name="city" placeholder="City"/>
+                        <label htmlFor="state">State</label>
+                        <input type="text" name="state" placeholder="State"/>
+                        <label htmlFor="description">Description</label>
+                        <textarea name="description" placeholder="Description"></textarea>
+                        <label htmlFor="campus">Campus Picture</label>
+                        <input type="file" name="campus" required/>
+                        <label htmlFor="logo">Logo</label>
+                        <input type="file" name="logo" required/>
```

Comment:
I think input names have to be `campusPic` and `universityLogo` for the form to be converted into a Chapter object backend.

Reply:
'campusPic' and 'universityLogo' don't work. I tested it and it is working with 'campus' and 'logo'. I believe it does this because it is written that way in the addChapter API when it calls the uploadImage function.